src/data/finset/basic.lean

```diff
@@ -1861,7 +1861,7 @@ lemma mem_map' (f : α ↪ β) {a} {s : finset α} : f a ∈ s.map f ↔ a ∈ s
 
 lemma mem_map_of_mem (f : α ↪ β) {a} {s : finset α} : a ∈ s → f a ∈ s.map f := (mem_map' _).2
 
-lemma forall_mem_map {f : α ↪ β} {s : finset α} {p : Π a, a ∈ s.map f → Prop} :
+lemma forall_map {f : α ↪ β} {s : finset α} {p : Π a, a ∈ s.map f → Prop} :
```

Comment:
@YaelDillies, the previous `forall_mem_map` was the name suggested by @urkud, and seems like the clearer name.
Can you justify why you think this name is better?

Reply:
This matches `set.ball_image_iff`, `set.exists_image_iff`, `set.forall_image2_iff`, `finset.forall_image₂_iff`...

Reply:
In `data.finset.basic` we use `finset.forall_mem_insert` etc. We should unify the naming convention some day but I think that consistency in one file is more important.

Reply:
`set.exists_image_iff` is about the subtype not membership, so isn't relevant here. I agree with @urkud, I think we should restore the mem name here (and personally, I think we should use either `forall_mem` everywhere or `ball` everywhere going forward).